notebooks/tropical/tropicalgemm.jl

```diff
@@ -211,30 +231,21 @@ md"e.g. create a strided pointer for an array"
 ptr = VectorizationBase.stridedpointer(v)
 
 # ╔═╡ 2afc083b-2db0-4401-a3d6-e259e5dee09d
-md"???"
+md"Offset a pointer:"
 
 # ╔═╡ b16fa1f0-466d-4ab1-aa3e-aaa98ec93e86
-VectorizationBase.gep(ptr.p, 1)
+VectorizationBase.gep(ptr, (5,)) == pointer(v) + sizeof(eltype(v))*(5-1)
 
 # ╔═╡ 2f0e159a-6b66-47c4-ac21-350ceeb3a5be
 md"""
-* `_vload` and `__vload` (loading data)
+* `vload` is for loading data
 """
 
-# ╔═╡ 9a7b58c9-cc12-4292-adb9-5538af4aa3a8
-VectorizationBase.vload(ptr, (3,))
-
-# ╔═╡ 9eebd69a-df78-47c2-827b-42691a1d7025
-md"e.g. load data into a 32*8 bit long register, and the offsets are (0, 8, 16, 24) bits, and mask out the first value."
-
 # ╔═╡ 7a80faf5-4fd5-4235-a14e-3e6482b57dfb
 vi = MM(StaticInt(4), StaticInt(0), StaticInt(8))
 
-# ╔═╡ f349285b-3374-4dde-9b6b-ee39bbb3b951
-md"Q: what is 8? why it is used as the normal stride in vload?"
-
 # ╔═╡ 2479b188-b242-487a-b08c-cde2d0d0468a
```

Comment:
Line 228-229 are for the following line rather than the previous line. Is this deleted by mistake?